app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -55,7 +55,7 @@ If you got your qualifications outside the UK, you should be able to request cop
 
 ### Lack of subject knowledge
 
-You may be able to do a [subject knowledge enhancement (SKE) course](/train-to-be-a-teacher/subject-knowledge-enhancement) to top up your subject knowledge. You could also find a course in a different subject.
+You could find a course in a different subject. If you're eligible, you might also be able to do a [subject knowledge enhancement (SKE) course](/train-to-be-a-teacher/subject-knowledge-enhancement) to top up your subject knowledge.
```

Comment:
Is it right to say 'also' here rather than 'or'? @philspray 
ie are the options that they could find a course in a different subject OR if eligible, do a SKE?

or are we saying they could find a course in a different subject and do a SKE for that?